be/app/Http/Controllers/LessonController.php

```diff
@@ -61,9 +63,20 @@ public function show($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
+    public function update(StoreLessonRequest $request, $id)
     {
-        //
+        $request->validated();
+
+        $lesson = Lesson::find($id);
+        $lesson->title = $request->title;
+        $lesson->description = $request->description;
+        $lesson->save();
+
+        return response([
+            'status' => 201,
```

Comment:
- follow proper format of status code on where to put status code (see my first comment).